api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -67,6 +85,8 @@ private Timestamp getCurrentTimestamp() {
 
   /** Makes a configurable number of project creation attempts. */
   public void bufferBillingProjects() {
+
+    monitoringService.sendLongSignal(BUFFER_ENTRY_MEASUREMENT, getCurrentBufferSize());
```

Comment:
For example, with an adapter service injected, you'd just have `billingProjectBufferMonitoringAdapter.logBufferSize(getBufferSize());`